xpdacq/xpdacq.py

```diff
@@ -198,7 +198,7 @@ def prun(sample,scanplan,**kwargs):
     if not dark_field_uid:
         dark_field_uid = dark(sample, scanplan, **kwargs)
```

Comment:
Based on my test at XPD, directly apply the sam scanplan object might lead to some problems. One possible situation is we will run a "dark" Tramp, if the there is no dark frame with the same exposure time as light frame.